### PR TITLE
Basic account viewer bounty

### PR DESCRIPTION
**Pull request by rahimklaber**
This is supposed to be a verry simple website to get people familiar with a web-wallet like albedo, freighter etc.

### CODE REVIEW

File: bounties/level-1/basic-account-viewer.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+## Description
+
+### What is this task?
+
+When building websites ontop of Stellar, there needs to be a way to access a user's Stellar account. This is where web-wallets like [Albedo](https://albedo.link/), [Rabet](https://rabet.io/) and [Freighter](https://www.freighter.app/) come in. They allow websites to easily and securely acces your Stellar account.
+
+Your task is to build a simple website which allows you to sign in using a web-wallet like Albedo, Freighter or Rabet.
+
+The website should show some basic information about the account currently logged in:
+ * The account's assets and their balances should be shown.
```
Comment (hanseartic) [inline bounties/level-1/basic-account-viewer.md:20]: Would you consider just showing basic information as asset-code and balance enough, as additional information is available from the well-know file?

File: bounties/level-1/basic-account-viewer.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ * The date when the account was created should be shown.
+ * The account who created this account should be shown.
+ 
+### What are the requirements for the bounty hunter?
+* The hunter should be able to build simple websites with Javascript.
+
+### What are the deliverables?
+#### Description
+  - The hunter should provide a link to the repository containing the solution.
```
Comment (hanseartic) [inline bounties/level-1/basic-account-viewer.md:29]: Wouldn't it be nice if the outcoume were some actually running website?

Easiest solution could be as gh-pages (e.g. https://github.com/gitname/react-gh-pages).

File: bounties/level-1/basic-account-viewer.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+#### Review Criteria
+  - As long as all features are present and correctly implemented, the submission should be accepted.
+
+
+## Links
+- [Albedo](https://albedo.link/)
+- [Rabet](https://rabet.io/)
+- [Freighter](https://www.freighter.app/)
+- [Stellar js Sdk](https://github.com/stellar/js-stellar-sdk)
```
Comment (hanseartic) [inline bounties/level-1/basic-account-viewer.md:39]:
```suggestion
- [Stellar js SDK](https://github.com/stellar/js-stellar-sdk)
```

File: bounties/level-1/basic-account-viewer.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+# Basic account viewer
+
+## Award
+| Hunter | Reviewer | Type
+| :- | :- | :-
+| 200 XLM | 1 Credits | Continuos
```
Comment (hanseartic) [inline bounties/level-1/basic-account-viewer.md:6]: Given the history of completed bounties I'd rather put this in the mid/upper L2 range.

File: bounties/level-1/basic-account-viewer.md (hunk 1 above)
Comment (rahimklaber) [reply bounties/level-1/basic-account-viewer.md:20]: Yes, I think it's enough to show the asset-code and balance. It's really simple, but I think thats find for this level. This is a bit unclear so I'll change it in the document.

File: bounties/level-1/basic-account-viewer.md (hunk 2 above)
Comment (rahimklaber) [reply bounties/level-1/basic-account-viewer.md:29]: Yes, that's a good idea.

File: bounties/level-1/basic-account-viewer.md (hunk 4 above)
Comment (rahimklaber) [reply bounties/level-1/basic-account-viewer.md:6]: I see what you mean. I think it's fine though, the acceptance criteria for this bounty is simple so I still think it should be level 1.

File: bounties/level-1/basic-account-viewer.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+ 
+### What are the requirements for the bounty hunter?
+* The hunter should be able to build simple websites with Javascript.
+
+### What are the deliverables?
+#### Description
+  - The hunter should provide a link to the repository containing the solution.
+
+#### Review Criteria
+  - As long as all features are present and correctly implemented, the submission should be accepted.
```
Comment (hanseartic) [inline bounties/level-1/basic-account-viewer.md:33]: I think it's a bit unclear what `features are present` means (e.g. range of possible solution mentioned in https://github.com/tyvdh/stellar-quest-bounties/pull/62/files#r682266645).

There could be requirements to
- choose an appropriate license
- necessity to have tests or not
- a hint about the UI/UX expectations to check
added

File: bounties/level-1/basic-account-viewer.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+## Description
+
+### What is this task?
+
+When building websites ontop of Stellar, there needs to be a way to access a user's Stellar account. This is where web-wallets like [Albedo](https://albedo.link/), [Rabet](https://rabet.io/) and [Freighter](https://www.freighter.app/) come in. They allow websites to easily and securely acces your Stellar account.
+
+Your task is to build a simple website which allows you to sign in using a web-wallet like Albedo, Freighter or Rabet.
+
+The website should show some basic information about the account currently logged in:
+ * The asset-codes and balances of the account's assets should be shown.
```
Comment (hanseartic) [inline bounties/level-1/basic-account-viewer.md:20]: Would you consider just showing basic information as asset-code and balance enough, as additional information is available from the well-know file?

File: bounties/level-1/basic-account-viewer.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
+# Basic account viewer
+
```
Comment (hanseartic) [inline bounties/level-1/basic-account-viewer.md:2]: We could have directly a link to claim, e.g.
```suggestion

[Claim](https://github.com/tyvdh/stellar-quest-bounties/issues/new?assignees=&labels=&template=begin-the-hunt.md&title=%F0%9F%94%B5+%60basic-account-viewer%60)  this bounty

```

File: bounties/level-1/basic-account-viewer.md (hunk 5 above)
Comment (rahimklaber) [reply bounties/level-1/basic-account-viewer.md:33]: Yeah you are right its unclear, I'll make it clearer for both the reviewer and hunter.

File: bounties/level-1/basic-account-viewer.md (hunk 7 above)
Comment (tyvdh) [reply bounties/level-1/basic-account-viewer.md:2]: Let's also add the View existing submissions bounty as outlined in this PR via this comment. https://github.com/tyvdh/stellar-quest-bounties/pull/63#discussion_r682795438

You can see what it looks like in the updated template.
https://github.com/tyvdh/stellar-quest-bounties/blob/main/bounties/README.md

File: bounties/level-1/basic-account-viewer.md (hunk 4 above)
Comment (tyvdh) [reply bounties/level-1/basic-account-viewer.md:8]: Let's leave this as L1 for now and see how it lands.

File: bounties/level-1/basic-account-viewer.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+[🔵 Start hunting](https://github.com/tyvdh/stellar-quest-bounties/issues/new?assignees=&labels=&template=begin-the-hunt.md&title=%F0%9F%94%B5+%60basic-account-viewer%60) this bounty
+
+## Award
+| Hunter | Reviewer | Type
+| :- | :- | :-
+| 200 XLM | 1 Credits | Continuos
```
Comment (tyvdh) [inline bounties/level-1/basic-account-viewer.md:8]: I would like the prize level to be 800 XLM and the review to be 2 Credits. This is a good bounty and I would like to see a lot of folks tackle this. Better to start too high and scale it back over time than too low and miss out on a great opportunity to attract interest.

File: bounties/level-1/basic-account-viewer.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+
+## Context
+https://github.com/tyvdh/stellar-quest-bounties/issues?q=basic-account-viewer.md
```
Comment (tyvdh) [inline bounties/level-1/basic-account-viewer.md:11]: See the new template and previous comments for how to format this now.

File: bounties/level-1/basic-account-viewer.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+
+## Description
+
+### What is this task?
+
+When building websites ontop of Stellar, there needs to be a way to access a user's Stellar account. This is where web-wallets like [Albedo](https://albedo.link/), [Rabet](https://rabet.io/) and [Freighter](https://www.freighter.app/) come in. They allow websites to easily and securely acces your Stellar account.
```
Comment (tyvdh) [inline bounties/level-1/basic-account-viewer.md:17]: acces -> access

File: bounties/level-1/basic-account-viewer.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+
+Your task is to build a simple website which allows you to sign in and log out using a web-wallet like Albedo, Freighter or Rabet.
```
Comment (tyvdh) [inline bounties/level-1/basic-account-viewer.md:19]: allows you -> allows users

File: bounties/level-1/basic-account-viewer.md [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+
+The website should show some basic information about the account currently logged in:
+ * The asset-codes and balances of the account's assets should be shown.
+ * The date when the account was created should be shown.
+ * The account who created this account should be shown.
```
Comment (tyvdh) [inline bounties/level-1/basic-account-viewer.md:24]: account who -> account which

File: bounties/level-1/basic-account-viewer.md [hunk 13]
```diff
@@ -0,0 +1,51 @@
...
+
+For this bounty, the UI is not important. However CSS and HTML should be used to create a simple layout. Additionally, the solution should be licensed as open source.
```
Comment (tyvdh) [inline bounties/level-1/basic-account-viewer.md:26]: the UI is not important -> the UI is not of primary importance

File: bounties/level-1/basic-account-viewer.md [hunk 14]
```diff
@@ -0,0 +1,51 @@
...
+### What are the requirements for the bounty hunter?
+* The hunter should be able to build simple websites with Javascript.
+
+### What are the deliverables?
+#### Description
+  - The hunter should provide a link to the repository containing the solution.
+  - The hunter should publish the website and provide a link to the published website.
+
+#### Review Criteria
+  - UI is not important, but there should atleast be a basic layout.
```
Comment (tyvdh) [inline bounties/level-1/basic-account-viewer.md:37]: UI is not or primary importance but there should at least be a basic layout.

File: bounties/level-1/basic-account-viewer.md [hunk 15]
```diff
@@ -0,0 +1,51 @@
...
+  - The code quality is not important.
```
Comment (tyvdh) [inline bounties/level-1/basic-account-viewer.md:38]: The code quality is also not primarily important but should function properly and be reasonably reviewable.

**Review by tyvdh — CHANGES_REQUESTED** [review]
Really nice bounty, great work on this. Just a couple tweaks and it's ready to merge!

**Pull request merged**